pyvrp/cpp/Individual_bindings.cpp

```diff
@@ -29,6 +29,15 @@ PYBIND11_MODULE(_Individual, m)
         .def("has_excess_capacity", &Individual::hasExcessCapacity)
         .def("has_time_warp", &Individual::hasTimeWarp)
         .def("__eq__", &Individual::operator==)
+        .def(
+            "__copy__",
```

Comment:
I forgot this in my earlier review, but can we add a test that checks copy.copy and copy.deepcopy actually return a copy (that is, == to original, but not when using `is`})?